cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -34,6 +34,52 @@ const (
 	statusPackage  = protogen.GoImportPath("google.golang.org/grpc/status")
 )
 
+type serviceGenerateHelper struct {
+	properties map[string]string
```

Comment:
What is this map used for?

Reply:
It records if the user enables renaming or not.

I record this information internally when create/override the helper:
`helper = &serviceGenerateHelperImpl{serviceGenerateHelper{properties: map[string]string{"rename": "false"}}}`

Using a map may look more generic than using a bool.

Reply:
You should be able to keep those implementation details out of OSS, I think.

Reply:
Done. Removed